libminifi/src/core/RepositoryFactory.cpp

```diff
@@ -48,9 +50,15 @@ std::shared_ptr<core::Repository> createRepository(
   try {
     std::shared_ptr<core::Repository> return_obj = nullptr;
     if (class_name_lc == "flowfilerepository") {
+      std::cout << "creating flow" << std::endl;
       return_obj = instantiate<core::repository::FlowFileRepository>();
     } else if (class_name_lc == "provenancerepository") {
       return_obj = instantiate<provenance::ProvenanceRepository>();
+    } else if (class_name_lc == "volatilerepository") {
+      return_obj = instantiate<repository::VolatileRepository>();
+    } else if (class_name_lc == "nooprepository") {
+      std::cout << "creating noop" << std::endl;
```

Comment:
it's needed for a test, but the test should be using the outputstreamappender. will fix. thanks.